src/main/webapp/css/home.css

```diff
@@ -1,9 +1,32 @@
-.body {
+body {
+  background-image: url("background2.jpeg");
+  background-size: cover;
+  font-family: Verdana, Geneva, sans-serif;
+  padding: 50px 60px;
   text-align: center;
 }
 
+.button{
+  background-color: rgb(16, 128, 148);
+  border: none;
+  color: white;
+  cursor: pointer;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.button:hover {
+  background-color: white;
+  color: rgb(16, 128, 148);
+  opacity: .5;
```

Comment:
I suggest this to be changed to 0.7 as 0.5 makes the button too blended in the background when hovering.

Reply:
Alright will do!